### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by malyniak**

### CODE REVIEW

File: src/BruteForce.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import java.util.*;
+public class BruteForce extends Decipher {
+    public Map<String, Key> getKeys() {
+        Map<String, Key> mapKeys = new TreeMap<>();
+        mapKeys.put("~~~", new KeyShift3(3));
```
Comment (Volodymyr95) [inline src/BruteForce.java]:
~~~
~~~~~
~~~~~~~~ make it constants

File: src/Caesar.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public abstract class Caesar {
+    Key key;
+    Path sourcePath;
+    Path outPath;
+    Menu menu=new Menu();
```
Comment (Volodymyr95) [inline src/Caesar.java]: Menu menu = new Menu();

File: src/Menu.java [hunk 3]
```diff
@@ -1,44 +1,87 @@
...
-    }
+        System.out.println(KEY_SELECT);
+        int k=Integer.parseInt(scanner.nextLine());
+        switch (k) {
+            case KEY3 -> {
 
+            return new KeyShift3(3);
+            }
+            case KEY5 ->
+            {
```
Comment (Volodymyr95) [inline src/Menu.java:72]: formatting

File: src/BruteForce.java (hunk 1 above)
Comment (malyniak) [reply src/BruteForce.java]: already removed the constructor with the parameter

File: src/Caesar.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+public abstract class Caesar {
+    Key key;
```
Comment (Volodymyr95) [inline src/Caesar.java]: why default access modifier?

File: src/Menu.java [hunk 5]
```diff
@@ -1,44 +1,81 @@
...
+            """
+                    Press 3 to move symbols 3 to the right
+                    Press 5 to move symbols 5 to the right
+                    Press 8 to move symbols 8 to the right
+                    """;
+    private static final int MENU_EXIT = 0;
+    private static final int MENU_ITEM0 = 1;
+    private static final int MENU_ITEM1 = 2;
+    private static final int MENU_ITEM2 = 3;
+    private static final int KEY3=3;
```
Comment (Volodymyr95) [inline src/Menu.java]: KEY_3
KEY_5
KEY_8

**Pull request merged**